ui/static/ui/js/learning_resources.jsx

```diff
@@ -465,7 +516,9 @@ define('learning_resources',
     getInitialState: function () {
       return {
         description: "",
+        descriptionCopy: "",
```

Comment:
Instead of `description` and `descriptionCopy` can you name these `originalDescription` and `description`? Otherwise it's not obvious which one a developer would use. It seems like the only use case of `originalDescription` is to compare with `description`, where `description` would be used everywhere else. Also rename `vocabulariesAndTerms` and `vocabulariesAndTermsCopy` the same way